controllers/config.go

```diff
@@ -0,0 +1,57 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+// images allows build systems to inject images for CCCMO components
+type images struct {
```

Comment:
You're shadowing the name of this struct in various places, perhaps call this `imageReference` and then it would be `images imgeReference` in other functions?

Reply:
I did the opposite :)